app/src/main/res/layout/downloads_sortby.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.mydownloads.downloads.DownloadsSortByViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="28dp"
+    android:layout_marginTop="28dp"
+    android:layout_marginEnd="28dp"
+    android:paddingBottom="28dp">
+
+    <TextView
+      android:id="@+id/sort_by_text_view"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginEnd="12dp"
+      android:fontFamily="sans-serif"
+      android:text="@string/sort_by"
+      android:textAllCaps="true"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="16sp"
+      app:layout_constraintBottom_toBottomOf="@+id/sort_by_container"
+      app:layout_constraintEnd_toStartOf="@id/sort_by_container"
+      app:layout_constraintHorizontal_bias="1.0"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="@+id/sort_by_container" />
+
+    <com.google.android.material.textfield.TextInputLayout
```

Comment:
This UI does not match with that of mocks in terms of `textSize`, `textColor`, `font`, `margins/paddings`, `iconColor`.
Can we fix this? Or is this the default UI similar to various AlertDialog box? If yer, then can you share some sample for references?

Reply:
fixed, done